tests/integration/test_host.py

```diff
@@ -44,95 +48,113 @@ async def hostprovider(adcm_client: ADCMClient, hostprovider_bundle: Bundle) ->
 
 
 async def test_host(adcm_client: ADCMClient, hostprovider: HostProvider, cluster: Cluster) -> None:
-    await _test_host_properties(adcm_client, hostprovider, cluster)
-    await _test_host_accessors(adcm_client, hostprovider, cluster)
-    await _test_pagination(adcm_client, hostprovider, cluster)
+    for i in range(55):
+        await adcm_client.hosts.create(
+            hostprovider=hostprovider,
+            name=f"test-host-{i}",
+        )
+
+    await cluster.hosts.add(host=await adcm_client.hosts.get(name__eq="test-host-0"))
+    await _test_host_properties(adcm_client.hosts, hostprovider, cluster)
+    await _test_host_accessors(adcm_client.hosts, cluster)
+    await _test_pagination(adcm_client.hosts)
+
+    for i in range(2, 55):
+        await cluster.hosts.add(host=await adcm_client.hosts.get(name__eq=f"test-host-{i}"))
+
+    await _test_pagination(cluster.hosts)
+    host = await adcm_client.hosts.get(name__icontains="T-10")
+    await cluster.hosts.remove(host)
+    await host.delete()
+
+
+async def test_host_in_host_group(adcm_client: ADCMClient, hostprovider: HostProvider, cluster: Cluster) -> None:
+    for i in range(80):
+        await adcm_client.hosts.create(name=f"test-host-{i}", hostprovider=hostprovider, cluster=cluster)
+
+    await cluster.config_host_groups.create(name="config-host-group", description="config host group description")
+    await cluster.action_host_groups.create(name="action-host-group", description="action host group description")
+
+    config_host_group = await cluster.config_host_groups.get(name__eq="config-host-group")
+    action_host_group = await cluster.action_host_groups.get(name__eq="action-host-group")
+    await config_host_group.hosts.add(await cluster.hosts.filter(name__contains="test-host"))
+    await action_host_group.hosts.add(await cluster.hosts.filter(name__contains="test-host"))
+
+    await _test_host_properties(config_host_group.hosts, hostprovider, cluster)
+    await _test_host_properties(config_host_group.hosts, hostprovider, cluster)
```

Comment:
Perhaps you had an action_host_group entry?

```suggestion
    await _test_host_properties(action_host_group.hosts, hostprovider, cluster)
```

Reply:
changed